src/renderers/WebGLRenderer.d.ts

```diff
@@ -350,7 +350,7 @@ export class WebGLRenderer implements Renderer {
 	 * properties to false. To forcibly clear one ore more buffers call {@link WebGLRenderer#clear .clear}.
 	 */
 	render(
-		scene: Scene,
+		scene: Scene | Object3D,
```

Comment:
Ah, it means it's sufficient to write:
```ts
scene: Object3D,
```

Reply:
Oh sorry I get it (Friday afternoon, right?). Will do.